browser/net/brave_ad_block_tp_network_delegate_helper.cc

```diff
@@ -245,6 +252,41 @@ void OnBeforeURLRequestAdBlockTP(const ResponseCallback& next_callback,
           brave_shields::features::kBraveAdblockCnameUncloaking) &&
       ctx->browser_context && !ctx->browser_context->IsTor();
 
+  // Also, skip CNAME uncloaking if there is currently a configured proxy.
+  if (ctx->browser_context) {
```

Comment:
I would extract this block to a separate function

Reply:
fixed!